airbyte-cdk/python/unit_tests/sources/file_based/scenarios/incremental_scenarios.py

```diff
@@ -1350,42 +1468,34 @@
         FileBasedSourceBuilder()
         .set_files(
             {
-                "b.csv": {
```

Comment:
Did this need to be reordered to make the test pass? IIRC there was at least one test where they were intentionally set out-of-order. I don't know if it was this specific test, but there was one that helped identify what would have been a bug in the order in which items get popped off of history.

Reply:
I didn't reorder anything, so this might have been a weird diff in the old incremental_scenarios to the new incremental_scenarios. I will take a further look to check though!